bindings/pydeck/examples/custom_layer.py

```diff
@@ -4,8 +4,8 @@
 
 pydeck.settings.custom_libraries = [
     {
-        "libraryName": "LabeledGeoJsonLayer",
-        "resourceUri": "https://raw.githubusercontent.com/ajduberstein/pydeck_custom_layer/master/dist/bundle.js",
+        "libraryName": "LabeledGeoJsonLayerLibrary",
+        "resourceUri": "https://unpkg.com/pydeck-custom-layer-demo@0.0.1/dist/bundle.js",
```

Comment:
Might be worth adding a comment here that the CDN needs to set CORS headers etc to enable the file to be loaded as as script. Not just any old CDN will work (e.g. per my experience, githubusercontent that you had here before won't work, but unpkg and jsdelivr will). 

It would also be good to mention this in any documentation on how to build, and load these modules.